src/client/menu/components/button/index.js

```diff
@@ -1,20 +1,37 @@
 import CSSModules from 'react-css-modules'
 import React from 'react'
 
-import { func, node } from 'prop-types'
+import { bool, func, node, oneOf, string } from 'prop-types'
 
 import css from './index.css'
 
+const TYPES = {
```

Comment:
Im not super opinionated on this, but wondering if there is a reason we aren't inlining these?